src/components/CheckboxGroup/index.d.ts

```diff
@@ -16,6 +16,7 @@ export interface CheckboxGroupProps extends BaseProps {
     error?: ReactNode;
     onChange?: (values: string[]) => void;
     id?: string;
+    orientation?: string;
```

Comment:
here we need to be a little bit more specific:
```orientation?: 'vertical' | 'horizontal'```

Reply:
Done https://github.com/nexxtway/react-rainbow/pull/1834/commits/58844df7f0f58f689898d98d6b10756f650c42bd#diff-2d2c8927e8faade40badec10e495da5d